content/spin/https-trigger.md

```diff
@@ -0,0 +1,55 @@
+title = "Exposing HTTP Triggers Using HTTPS"
+template = "spin_main"
+date = "2022-02-01T00:00:00Z"
+[extra]
+
+---
+- [HTTP vs HTTPS](#http-vs-https)
+- [Routing](#routing)
+- [Defining the HTTP Object](#defining-the-http-object)
+
+## HTTP vs HTTPS
+
+The [Spin HTTP trigger document](./http-trigger.md) provides a great overview of the HTTP trigger, as well as implementation details around the WebAssembly component model and how it
+is used in Spin. However, there may be certain situations where you would want to accept only HTTPS requests (and reject HTTP requests). This raises questions like, do I have to provide a TLS certificate, and so forth?
+
+As mentioned in the original HTTP trigger article, your HTTP application is [configured](./configuration.md) by defining a top-level application trigger. For example:
+
+<!-- @nocpy -->
+
+```toml
+# spin.toml
+trigger = { type = "http", base = "/" }
+```
+
+In addition to the above configuration, we also create the `route` and `executor` (which both come under the `[component.trigger]` section, as shown below):
+
+<!-- @nocpy -->
+
+```toml
+[component.trigger]
+route = "/hello"
+executor = { type = "spin" }
+```
+
+## Routing
+
+If you are curious about routing (or need a refresher) please see the [routing section in the HTTP trigger document](./http-trigger#routing).
+
+## Defining the HTTP Object
+
+We define the HTTP objects as [WebAssembly Interface (WIT)](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md) and therefore one could go ahead and create a `SPIN_TLS_CERT` environment variable and a `SPIN_TLS_KEY` environment variable. For example:
+
+<!-- @nocpy -->
+
+```bash
+export SPIN_TLS_CERT=publickey.pem
+```
+
+<!-- @nocpy -->
+
+```bash
+export SPIN_TLS_KEY=pkcs8.key
+```
+
+Once created, both of the above environmental variables can be used as arguments of the `spin up` command.
```

Comment:
This is a bit confusing - do you mean I have to pass these to `spin up` as command line options?  And if so how?  (The answer is no, if they are set when you run `spin up` then Spin will pick them up automatically.  You can pass by arguments _instead_ using the `--tls-*` command line options.)